proms-resources/frontend/src/main/frontend/src/proms/QuestionnaireSet.jsx

```diff
@@ -214,13 +216,25 @@ function QuestionnaireSet(props) {
     }
   }, [subjectDataLoadCount]);
 
-  // When the user lands on a completed visit that has not been submited, proceed to reviewing their forms
+  // When the user lands on a completed visit that has not been submitted, proceed to reviewing their forms
   useEffect(() => {
     if(isComplete && !isSubmitted && questionnaireIds?.length > 0 && crtStep == -1) {
       setCrtStep(questionnaireIds.length)
     }
   }, [isComplete, isSubmitted])
 
+  // Fetch saved settings for Patient Portal Survey Instructions
+  useEffect(() => {
+    fetch('/Proms/SurveyInstructions.json')
+      .then((response) => response.ok ? response.json() : Promise.reject(response))
+      .then((json) => {
+        setSurveyInstructions(json);
```

Comment:
This is where we should fill in the gaps with the default instructions.
```suggestion
        setSurveyInstructions(Object.assign(defaultInstructions, json));
```